doc/src/manual/interfaces.md

```diff
@@ -408,6 +408,7 @@ perhaps range-types `Ind` of your own design. For more information, see [Arrays
 |:----------------------------------------------- |:-------------------------------------- |:------------------------------------------------------------------------------------- |
 | `strides(A)`                             |                                        | Return the distance in memory (in number of elements) between adjacent elements in each dimension as a tuple. If `A` is an `AbstractArray{T,0}`, this should return an empty tuple.    |
 | `Base.unsafe_convert(::Type{Ptr{T}}, A)`        |                                        | Return the native address of an array.                                            |
+| `Base.MemoryLayout(A)`                 |                                        | Return a subtype of `LinearAlgebra.AbstractStridedLayout`,  such as `LinearAlgebra.DenseColumnMajor{T}()`,`LinearAlgebra.DenseRowMajor{T}()`, or `LinearAlgebra.StridedLayout{T}()`.
```

Comment:
I don't think `LinearAlgebra.AbstractStridedLayout` should be in LinearAlgebra; is this a typo?

Reply:
Fixed.